pulp_rpm/app/kickstart/treeinfo.py

```diff
@@ -22,14 +21,14 @@ def get_treeinfo_data(remote, remote_url):
     treeinfo_serialized = {}
     namespaces = [".treeinfo", "treeinfo"]
     for namespace in namespaces:
-        downloader = remote.get_downloader(url=urljoin(remote_url, namespace))
+        downloader = remote.get_downloader(
+            url=urljoin(remote_url, namespace), silence_errors_for_response_status_codes={404}
+        )
 
         try:
             result = downloader.fetch()
-        except ClientResponseError as exc:
-            if 404 == exc.status:
+            if getattr(downloader, "response_status", 0) == 404:
```

Comment:
Ref: https://github.com/pulp/pulpcore/pull/810
when skipping 404, it downloads the 404 page, when 
```python
treeinfo.load(f=result.path)
```
it fails because the downloaded file is not `INI` format